deps/v8/test/mjsunit/es6/typed-array-iterator.js

```diff
@@ -21,9 +21,9 @@ function TestTypedArrayPrototype(constructor) {
   assertFalse(constructor.prototype.propertyIsEnumerable(Symbol.iterator));
 
   assertEquals(Array.prototype.entries, constructor.prototype.entries);
-  assertEquals(Array.prototype[Symbol.iterator], constructor.prototype.values);
+  assertEquals(Array.prototype.values, constructor.prototype.values);
```

Comment:
I simply did a revert on the commit that removed `Array.prototype.values()`. What the v8 team does with their tests is their own business :-)